recipe-client-addon/test/browser/browser_RecipeRunner.js

```diff
@@ -225,6 +226,7 @@ add_task(async function testStartup() {
 
   // not in dev mode
   await SpecialPowers.pushPrefEnv({set: [["extensions.shield-recipe-client.dev_mode", false]]});
+  await SpecialPowers.pushPrefEnv({set: [["extensions.shield-recipe-client.first_run", false]]});
   RecipeRunner.init();
   ok(!runStub.called, "RecipeRunner.run is not called immediately when not in dev mode");
   ok(addCleanupHandlerStub.called, "A cleanup function is registered when not in dev mode");
```

Comment:
I think this is impossible to test well with browser mochitests. You may stand a chance using marionette, but equally I don't think that needs to block landing.